res/css/views/settings/_NotificationSettings2.pcss

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_NotificationSettings2 {
+    .mx_SettingsSection_subSections {
+        color: $primary-content;
+        gap: 32px;
+        display: flex;
+        flex-direction: column;
+    }
+
+    .mx_SettingsSubsection_description {
+        margin-bottom: 20px;
+
+        .mx_SettingsSubsection_text {
+            font-size: 1.2rem;
```

Comment:
Why is this here? It makes the descriptions look rather silly:
![image](https://github.com/matrix-org/matrix-react-sdk/assets/3107142/4071517b-2339-4263-a074-36e101ac0d15)